XiEditor/Search.swift

```diff
@@ -171,6 +171,10 @@ extension EditViewController {
         findViewController.replacePanel.isHidden = replaceHidden
         let offset = findViewController.view.fittingSize.height
         scrollView.contentInsets = NSEdgeInsetsMake(offset, 0, 0, 0)
+
+        let origin = scrollView.visibleRect.origin
```

Comment:
`scrollView.visibleRect` is just the frame of the scrollView; you probably want `scrollView.contentView.visibleRect`?

As is this only works at the top of the file.

Reply:
Fixed